scripts/fuzz_opt.py

```diff
@@ -35,9 +35,14 @@
 
 # parameters
 
+NOMINAL = True
```

Comment:
Are we ready for something like this?

```python
# --structural, --nominal, or --hybrid
TYPE_SYSTEM = '--nominal'
```

Reply:
Sounds good to generalize, done. That should be helpful later. (I haven't done any hybrid fuzzing yet, though.)